crates/modelardb_server/src/storage/uncompressed_data_manager.rs

```diff
@@ -78,7 +79,7 @@ impl UncompressedDataManager {
         local_data_folder: PathBuf,
         memory_pool: Arc<MemoryPool>,
         channels: Arc<Channels>,
-        metadata_manager: Arc<MetadataManager>,
+        metadata_manager: Arc<TableMetadataManager<Sqlite>>,
```

Comment:
Maybe renamed to `table_metadata_manager` for consistency.

Reply:
Fixed.